t/099-c-api.t

```diff
@@ -229,9 +229,8 @@ bar: rc=0, type=1, val=0
     }
 --- request
 GET /test
---- response_body
-foo: rc=-5
-bar: rc=-5
+--- response_body eval
+qr/foo: rc=-5\nbar: rc=-5\n|bar: rc=-5\nfoo: rc=-5\n/
```

Comment:
Yes, you are right. In a hindsight I do not know why I added this. I had problems with ffi.